### PR TITLE
Release Flux v1.10.1

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+### Improvements
+
+- Support Azure DevOps Git hosts
+  [weaveworks/flux#1729][#1729]
+  [weaveworks/flux#1731][#1731]
+- Use AKS credentials for ACR
+  [weaveworks/flux#1694][#1694]
+- Make port forward label selector configurable
+  [weaveworks/flux#1727][#1727]
+- Add image repo and pull secret to chart readme
```
Comment (hiddeco) [inline CHANGELOG.md]: Helm chart change, not related to Flux (daemon).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+  [weaveworks/flux#1721][#1721]
+- Add "skipDepUpdate" config parameter to HelmRelease
```
Comment (hiddeco) [inline CHANGELOG.md]: Helm operator change, not related to Flux (daemon).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,40 @@
...
+  [weaveworks/flux#1712][#1712]
+- Add image pull secret options to Helm chart
```
Comment (hiddeco) [inline CHANGELOG.md]: Helm chart change, not related to Flux (daemon).

**Comment by 2opremio** [comment]
> Apart from the comments below I also change the tag in the manifest (`deploy/flux-deployment.yaml`) to the upcoming version so that `master` contains the latest release information when `release/10.0.x` is merged back into it.

Good one., done.

This is something we should probably automate. For starters I will add it to the release docs.

**Comment by hiddeco** [comment]
> This is something we should probably automate.

Agreed, another thing on my list to automate is the ToC we have on some doc pages as the formatting of the URLs (and keeping them up to date) is tricky, plus it results in stuff like #1731.

**Pull request merged**